src/libYARP_dev/src/IVisualParamsImpl.cpp

```diff
@@ -202,12 +208,11 @@ bool Implement_RgbVisualParams_Parser::respond(const yarp::os::Bottle& cmd, yarp
                     ret = iRgbVisual->getRgbIntrinsicParam(params);
                     if(ret)
                     {
-                        yarp::os::Bottle params_b;
                         response.addVocab(VOCAB_RGB_VISUAL_PARAMS);
                         response.addVocab(VOCAB_INTRINSIC_PARAM);
                         response.addVocab(VOCAB_IS);
-                        ret &= Property::copyPortable(params, params_b);  // will it really work??
-                        response.append(params_b);
+                        yarp::os::Bottle& tmp=response.addList();
+                        ret &= Property::copyPortable(params, tmp);
```

Comment:
I see, I probably saw it before and was confused as well, but then I forgot about it. 
If it is so widespread, then it definitely does not make sense to change just this one in this PR. 

Just one additional comment for clarify for other people reading this: `a &= b` **is not** equivalent to `a = a && b`. In particular, if b is a function call `f()` (as in the case of this PR) then in `a &= f()` `f()` is always called regardless of the value of `a`, while in `a = a && f()` `f()` is not called if `a` is false. This kind of tricky behavior are one of the reasons I personally always prefer a readable alternative to such terse statements. In particular I prefer forms for which no programmer, regardless of their level of knowledge of the C++ standard, can get confused, such as:
~~~
bool copySuccessfull = Property::copyPortable(params, tmp);
ret = ret && copySucessfull;
~~~